model/sshable.rb

```diff
@@ -138,6 +138,13 @@ def invalidate_cache_entry
     Thread.current[:clover_ssh_cache]&.delete([host, unix_user])
   end
 
+  def available?
+    cmd("true")
+    true
+  rescue
```

Comment:
also, blanket rescue is probably no what I'd do here, it can pick up a lot of stuff. 

We have a laundry list of ioerror-and-similar in host_nexus.rb:

`    rescue Net::SSH::Disconnect, Net::SSH::ConnectionTimeout, Errno::ECONNRESET, Errno::ECONNREFUSED, IOError`

So how about this: there's probably something here to make life a bit better, by encapsulating the "IOError and Friends" list, by using a block or possibly a flag to cmd. Maybe that can square the circle? The other reason I can think to do what you are doing is this `rescue` making the caller simpler.

Reply:
>The other reason I can think to do what you are doing is this rescue making the caller simpler.

That's correct, also I thought this was similar to `HostNexus#available`. 

I am taking a stab now to make it more elegant, but if I can't come up with an good solution to wrap "expected exceptions", I will revert this part and handle the exception in the caller.

Reply:
I made the rescue catch only the "IOerror and friends", also modified the code in hostnexus to separate availability check. Not sure if that's good enough, if not jut LMK and I will remove the available? function.